scrypt.h

```diff
@@ -7,7 +7,7 @@ extern "C" {
 
 void scrypt_1024_1_1_256(const char* input, char* output);
 void scrypt_1024_1_1_256_sp(const char* input, char* output, char* scratchpad);
-const int scrypt_scratchpad_size = 131583;
+//const int scrypt_scratchpad_size = 131583;
```

Comment:
Tested on x86_64-pc-linux-gnu and this line does nothing either way. What was reason for commenting out?

Reply:
I think this is because for some reason, 131583 is hard-coded and the size variable isn't used:

https://github.com/dogecoin/ltc-scrypt/blob/36f7d5b74d5823d205670b2d26857c7e81279606/scrypt.c#L678

Reply:
Ah yes I see it way down at the bottom. Thanks!